src/hackerspace_online/management/commands/lowercase_usernames.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from django.db import models
+from django.db.models.functions import Lower
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+
+    help = 'Used to lowercase usernames'
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+
+        # First let's take all users that have 0 duplicates
+        users = User.objects.values('username').annotate(existing_count=models.Count(Lower('username')))
+
+        unique_usernames = users.filter(existing_count=1).values_list('username', flat=True)
+
+        # Update their usernaems to lowercase
+        count = User.objects.filter(username__in=unique_usernames).update(username=Lower('username'))
+        self.stdout.write(self.style.SUCCESS(f'Updated {count} users!\n'))
```

Comment:
Let's stick with print() for this project, self.stdout.write adds nothing and is ugly =)

Reply:
https://docs.djangoproject.com/en/3.2/howto/custom-management-commands/

It's being recommended from the django docs.

> Note
> 
> When you are using management commands and wish to provide console output, you should write to self.stdout and self.stderr, instead of printing to stdout and stderr directly. By using these proxies, it becomes much easier to test your custom command. Note also that you don’t need to end messages with a newline character, it will be added automatically, unless you specify the ending parameter:


Should we just use print then?

Reply:
Nope, good catch.  Let's stick with the recommendation in the docs.  In that case, you still have one `print()` in there that should be changed to self.stdout.write! =D